scripts/generate_genotype_df/README.md

```diff
@@ -0,0 +1,9 @@
+# Export TOB-WGS dataset to PLINK format
+
+This runs a Hail query script in Dataproc using Hail Batch in order to export the TOB-WGS genotypes, which are requirements for the association analysis. Only variants that pass VQSR and GQ filters are retained, as well as variants with an MAF > 0.01. To run, use conda to install the analysis-runner, then execute the following command:
```

Comment:
Nit: the analysis-runner is now installed using pip.

Reply:
Fixed